app/tests/src/transaction-planner.js

```diff
@@ -0,0 +1,65 @@
+import assert from 'assert'
+import {selectMinimalTxPlan} from '../../frontend/wallet/shelley/shelley-transaction-planner'
+import {AssetFamily, TxType} from '../..//frontend/types'
+
+const utxos = {
+  utxo1: {
+    txHash: 'deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef',
+    address:
+      'addr1q8eakg39wqlye7lzyfmh900s2luc99zf7x9vs839pn4srjs2s3ps2plp2rc2qcgfmsa8kx2kk7s9s6hfq799tmcwpvpsjv0zk3',
+    coins: 1000000,
+    outputIndex: 1,
+    tokenBundle: [],
+  },
+  utxo2: {
+    txHash: 'deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef',
+    address:
+      'addr1q8eakg39wqlye7lzyfmh900s2luc99zf7x9vs839pn4srjs2s3ps2plp2rc2qcgfmsa8kx2kk7s9s6hfq799tmcwpvpsjv0zk3',
+    coins: 2000000,
+    outputIndex: 0,
+    tokenBundle: [],
+  },
+}
+
+const successPlanSettings = {
```

Comment:
I'd name this successPlanFixtures, which is a more standard terminology for tests